core-primitives/ocall-api/src/lib.rs

```diff
@@ -17,13 +17,15 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+pub extern crate alloc;
+
+use alloc::vec::Vec;
 use codec::{Decode, Encode};
 use core::fmt::Debug;
 use itp_types::{TrustedOperationStatus, WorkerRequest, WorkerResponse};
 use its_primitives::traits::SignedBlock;
 use sgx_types::*;
 use sp_runtime::OpaqueExtrinsic;
-use sp_std::prelude::Vec;
```

Comment:
Here I'm not sure: We have 3 ways to bring some of the `alloc` or `std` types into our `sgx` feature:
* `extern crate alloc` + `use alloc::vec::Vec`
* `use std::vec::Vec`
* `use sp_std::prelude::Vec` (re-export in `sp_std`)

What do you guys think? Which option do you prefer and why?